.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: maven
+    directory: "/"
+    schedule:
+      interval: monthly
```

Comment:
There's no reason to enable Dependabot if nobody cares about the PRs it opens, they just clutter up the repo.